src/main/java/com/salesforce/dataloader/security/EncryptionUtil.java

```diff
@@ -98,28 +115,27 @@ private static String bytesToText(byte[] bytes) {
             }
             sb.append(hex);
         }
-
         return sb.toString();
     }
 
+
     /**
      * Create a key for encryption and decryption
      *
      * @return Key
-     * @throws NoSuchAlgorithmException
-     * @throws InvalidKeySpecException
-     * @throws InvalidKeyException
```

Comment:
You still have these exceptions as throws, but why removed them from comments?

I see that all @throws and @return in comments are removed in this CL. Any particular reason?

Reply:
Using throws and returns are not recommended any more.